pkg/apis/core/v1alpha1/types_propagationpolicy.go

```diff
@@ -106,7 +107,12 @@ type PropagationPolicySpec struct {
 	// +optional
 	// Default set via a post-generation patch.
 	// See patch file for details.
+	// Deprecated: Please use reschedulePolicy.replicaRescheduling instead, and it will be removed in next release.
 	ReplicaRescheduling *ReplicaRescheduling `json:"replicaRescheduling,omitempty"`
+
+	// Configures behaviors related to rescheduling.
+	// +optional
+	ReschedulePolicy ReschedulePolicy `json:"reschedulePolicy"`
```

Comment:
Should this be a pointer field?

Reply:
At least there must be a value, so there is no need for a pointer?